client/app/services/contactEditor.js

```diff
@@ -1,9 +1,9 @@
 angular.module('distress')
 
-.factory('ContactEdit', function($http){
-  var contactEdit = {};
+.factory('ContactEditor', function($http){
+  var contactEditor = {};
```

Comment:
var contactEditor = {};

can it be 'var instance = {}'? that's what we used in the other factories